webapp/src/Entity/ContestProblem.php

```diff
@@ -189,13 +189,13 @@ public function getColor(): ?string
         return $this->color;
     }
 
-    public function setLazyEvalResults(?bool $lazyEvalResults): ContestProblem
+    public function setLazyEvalResults(?int $lazyEvalResults): ContestProblem
     {
-        $this->lazyEvalResults = $lazyEvalResults;
+        $this->lazyEvalResults = $lazyEvalResults === 0 ? null : $lazyEvalResults;
```

Comment:
Why do we need to check for `=== 0` here?

Reply:
The POST sends a 0, but I wanted to store this as a null to have a clear difference between a default value and a value which is explicit set. I can change this to just be 0 now that everything is annotated with constants so it is easier to read in code if that is preferred.